.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install build dependencies
-      run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov
+      run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen default-jdk-headless
```

Comment:
I guess we want to run these new tests for a while. Probably, better to use daily for it, not to slow-down CI. 
We can have a job there just for elle. Similar to `test-macos`, `test-tls-sanitizer`, we can have `test-elle`.

Reply:
created an elle test suite for daily